OPENSSL.md

```diff
@@ -0,0 +1,77 @@
+### Building OpenSSL for shell bundles
+
+To build OpenSSL for shell bundles, we require building:
```

Comment:
I like your write-up for describing a shell bundle, but won't incorporate it here. Here, I'll change occurrences of "shell bundle" with "universal agent".

You can build universal agent builds without a shell bundle, and still need OpenSSL binaries.